core/test-extension/runtime/src/main/java/io/quarkus/extest/runtime/config/AllValuesConfig.java

```diff
@@ -30,7 +29,7 @@
     public OptionalDouble optDoubleValue;
     /** an optional long value */
     @ConfigItem
-    public Optional<Long> optionalLongValue;
```

Comment:
This should not be changed: it is like this deliberately in order to test `Optional`.  If we change it to `OptionalLong` then we won't be testing that anymore.